### ISSUE TITLE
Radio buttons display and functionality broken

### ISSUE THREAD

**Issue by biki** [wagtail/wagtail-react-streamfield#36]
Wagtail 2.4
Wagtail react streamfield 1.2.0

Radio buttons within the edit form don't have enough space and grow into each other. Also they look like they're not properly given a name attribute which is why you're able to select more than one.

![image](https://user-images.githubusercontent.com/516766/54107065-5d5ec680-43d8-11e9-925e-8501c3dc250d.png)

Note that radio buttons are not default Wagtail, but a customized ChoiceBlock:
![image](https://user-images.githubusercontent.com/516766/54107382-42408680-43d9-11e9-8e44-5b8bc76ca27b.png)

**Comment by BertrandBordage** [comment]
Half of that issue was fixed by https://github.com/wagtail/react-streamfield/commit/e775cd303323daabc6de824d23b1d74d6cb35354.
That fix is for the exclusive selection of a single radio button at a time.

What remains is to fix the CSS issue, it’s purely wagtail related.

**Comment by BertrandBordage** [comment]
The style issue was fixed by 45dcaf232020c0adc8af7db5917536d9b1c0de82, and the react-streamfield fix is in the version used by wagtail-react-streamfield now.

So everything here is fixed, and will be in the new wagtail-react-streamfield version :)

**Comment by biki** [comment]
Unfortunately I have to open this issue again, it is still somewhat broken in 1.3.0

<img width="370" alt="Screenshot 2019-08-08 at 08 32 37" src="https://user-images.githubusercontent.com/516766/62680271-1ee52c00-b9b7-11e9-895e-1868c8d5e736.png">

Getting the following error:

<img width="883" alt="Screenshot 2019-08-08 at 08 33 41" src="https://user-images.githubusercontent.com/516766/62680317-420fdb80-b9b7-11e9-84ac-62f9ceff18a4.png">

If you need more information from me, please say so. Thank you.

**Comment by BertrandBordage** [comment]
@biki Did you try force-refreshing the browser? With `⌘`+`shift`+`R` on Apple computers and `ctrl`+`F5` on everything else.

**Comment by biki** [comment]
Yep. I know that Wagtail sometimes bugs out after updating but I've cleared the cache multiple times.

**Comment by BertrandBordage** [comment]
Was it using the exact `RadioBlock` you defined above? I’m asking because that’s what I used for testing, and it was working flawlessly.

**Comment by biki** [comment]
Yep, using the same definition on Wagtail 2.6.1
I'm using `RadioBlock` like this:
```
image_text_cubes = ('image_text_cubes', blocks.StructBlock([
    ('title', blocks.CharBlock(required=False)),
    ('cubes', blocks.ListBlock(blocks.StructBlock([
        ('image', ImageChooserBlock()),
        ('image_position', RadioBlock(choices=DEFAULT_ALIGNMENT_CHOICES[:2], default='left')),
        ('title', blocks.CharBlock()),
        ('content', blocks.RichTextBlock()),
        ('list_style', RadioBlock(choices=DEFAULT_LIST_CHOICES, default='dash')),
        ('call_to_actions', blocks.ListBlock(CallToActions(), default=[], required=False, null=True, blank=True)),
    ]))),
], template='content/blocks/image_text_cubes.html', icon='fa-cubes', group=GROUP_NAME))```

**Comment by BertrandBordage** [comment]
OK, I can confirm I see a buggy behaviour with `RadioBlock` after values were saved with the old StreamField. I’m investigating it.

**Comment by BertrandBordage** [comment]
That last issue was fixed by df10ce3caeab994cb2864b18d611ef3cdfe37ff6.
It was a bad radio serialization issue. I discovered another issue at the same time, one method was not correctly monkey patched, so the error messages were no longer shown in the field blocks.

That should fix your issue, tell me otherwise!

**Comment by BertrandBordage** [comment]
I forgot to tell you, it’s in a new 1.3.1 release.
